server/controllers/userFollow.js

```diff
@@ -0,0 +1,132 @@
+import models from "@models";
+import { serverError, httpResponse } from "@helpers/http";
+
+const { Followers, Users } = models;
+
+/**
+ * @description Follow or unfollow a user
+ *
+ * @param {object} req HTTP request object
+ * @param {object} res HTTP response object
+ * @returns {object} HTTP response
+ */
+export const followOrUnfollowUser = async (req, res) => {
+  const {
+    userDetails,
+    user: { userId }
+  } = req;
+  const authorId = userDetails.id;
+  const author = await userDetails.getFullName();
+  try {
+    if (userId === authorId) {
+      return httpResponse(res, {
+        statusCode: 403,
+        success: false,
+        message: "You cannot follow yourself"
+      });
+    }
+    const follow = await Followers.findOrCreate({
+      where: { authorId, followerId: userId }
+    });
+    const following = follow[1];
+    if (following === true) {
```

Comment:
You don't need this true here you can say if(following)

Reply:
Implemented!